contracts/infinite-pool/RewardsPoolBaseInfinite.sol

```diff
@@ -0,0 +1,566 @@
+// SPDX-License-Identifier: MIT
```

Comment:
There is no way to withdraw excess rewards send to the contract

Reply:
I again some functionality for this, I forgot to migrate this from the current rewardsPoolBase